### ISSUE TITLE
Synop not working anymore after latest update from KMI

### ISSUE THREAD

**Issue by TimFranken** [TimFranken/openkmi#11]
Synop is not working anymore after latest update. Error related to the `code` field that's not in the response anymore.

**Comment by TimFranken** [comment]
Closed by f6e92681e00f14c2b70fe6565bba5fdab866cf5a

**Comment by linusderoo** [comment]
They changed it again so the 'code' field is back in the output. Replacing line 30 with `df_stations['code'] = df_stations['code'].apply(str)` works.

**Comment by TimFranken** [comment]
Hi @linusderoo , thanks for the comment. Does that mean the synop code doesn't work anymore? (Currently traveling so can't properly test for the moment).

**Comment by linusderoo** [comment]
Correct, it throws an error since the FID column doesn't contain the 'station code' anymore. Replacing the line as mentioned above works for me.

**Comment by TimFranken** [comment]
Closed by ba28bf3c49fc7f5d98728679e5dd4cf3b4b568be
